### DISCUSSION TITLE
Discussions

### DISCUSSION THREAD

**Discussion by danstis** [DayZTools/dayz-server-template#4] [General]
Discussions are in use, feel free to ask questions etc using the discussions in this repo... 
If you find issues then use the issues tab above to raise them with as much detail as possible. :)

**Comment by starswift42** [comment]
Hi @danstis First of all, thank you. I was looking for this exact solution for the better part of a year.  Secondly, I wondered if you were still using this method and if so, have their been any updates to it?

**Reply by danstis** [reply]
Hey, are you referring to the pipeline to upload the files to Nitrado on save?
If yes, then I still use this solution on many of the servers that I am involved with. There are a couple of small updates will I will merge in today to bring this up to my current pipeline.

Also note that the wiki has some instructions on how the pipeline is configured. https://github.com/DayZTools/dayz-server-template/wiki

**Reply by danstis** [reply]
I have merged in the updates now. So this is the latest version that I am running.
